staging/src/github.com/kubeedge/mapper-framework/README.md

```diff
@@ -42,6 +42,30 @@ mapper
 ├── pkg ------------------------ Mapper register process, almost need not change
 └── Makefile
 ```
+In most cases, you need to fill in the code in the driver folder as follows: In the devicetype.go file, 
+you need to fill in the ProtocolConfig and VisitorConfig structure information in the way you defined in the instance yaml file, 
+so that the mapper can correctly parse the configuration information. In the driver.go file, 
+you need to customize the method of initializing the device and obtaining device data, and standardize 
+the data collected by the mapper. In config.yaml, you need to define the protocol name of mapper.
+## 2. Generate the mapper project
+After generating the mapper project and filling driver folder, users can make their own mapper image 
+based on the Dockerfile file and deploy the mapper in the cluster through deployment and other methods.
+```shell
+  docker build -t [YOUR MAPPER IMAGE NAME] .
+```
+If you want to debug locally first, you can also compile and run the mapper code directly:
+```shell
+  go run -tags Nostream cmd/main.go --v <log level,like 3> --config-file <path to config yaml>
+```
 
+The above operations are mainly for mappers that do not process streaming data. If you need to enable 
+the streaming data processing function, please execute the following command:
+```shell
+  go run -tags stream cmd/main.go --v <log level,like 3> --config-file <path to config yaml>
+```
```

Comment:
It is not recommended to write too much here, just write it in the document.

Reply:
I've simplified the README file but still kept some necessary steps to use it